### PR TITLE
Update poetry lock

### PR DESCRIPTION
**Pull request by lopagela**
Updated `llama-index` to `0.8.67` to avoid users to have issues around `openai` import.

Re-generated the poetry.lock file by running:

```bash
rm -rf .venv/ poetry.lock

python3 -m venv .venv && source .venv/bin/activate && pip install --upgrade pip poetry && poetry install --with ui,local
```

Tested locally, and the basic call through the UI works as expected.

### CODE REVIEW

File: settings.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ ui:
   path: /
 
 llm:
-  mode: mock
+  mode: local
```
Comment (lopagela) [inline settings.yaml:18]: Noteworthy change

**Review by pabloogc — APPROVED** [review]

**Review by imartinez — APPROVED** [review]

**Comment by pabloogc** [comment]
looks like the bump to llama Index broke a type

**Comment by lopagela** [comment]
> looks like the bump to llama index broke a type

@pabloogc , indeed, thank you, will fix (this is a warning that have been transformed into an error apparently :sweat_smile: -- `enum` in `Field` is not authorized anymore)

Fixing

**Comment by lopagela** [comment]
@pabloogc fixed 

As explained in the commit message, I had to fight with function signature and do one extra `cast`

**Review by pabloogc — APPROVED** [review]

**Pull request merged**